File: 0162-find-peak-element/0162-find-peak-element.cpp
class Solution {
public:
    int findPeakElement(vector<int>& nums) {
        int max_idx = 0;

        // 1. Changed .length() to .size()
        for(int i = 1; i < nums.size(); i++) {
            if(nums[i] > nums[i-1]) {
                // 2. Changed Math.max to std::max
                max_idx = std::max(max_idx, i);
            }
        }        
        return max_idx;
    }
};

File: 0162-find-peak-element/0162-find-peak-element.java
class Solution {
    public int findPeakElement(int[] nums) {

        if(nums.length == 1) return 0;

        int max = 0;

        for(int i = 1; i < nums.length; i++){
            if(nums[i]> nums[i-1]){
                

                max = Math.max(max, i);
            }
        }
        return max;
    }
}